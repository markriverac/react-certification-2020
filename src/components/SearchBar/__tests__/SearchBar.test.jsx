import React from 'react';
import renderer from 'react-test-renderer';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Searchbar from '../SearchBar.component';

Enzyme.configure({ adapter: new Adapter() });

it('Renders button without exploding', () => {
  const video = {};
  const searchBar = renderer.create(<Searchbar video={video} />);
  const tree = searchBar.toJSON();
  expect(tree).toMatchSnapshot();
});
