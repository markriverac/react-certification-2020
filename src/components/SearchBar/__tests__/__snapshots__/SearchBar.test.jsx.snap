// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders button without exploding 1`] = `
<div>
  <form
    className="ui form"
    onSubmit={[Function]}
  >
    <div
      className="field"
    >
      <input
        className="main-search"
        name="video-search"
        onChange={[Function]}
        placeholder="Tell me what you need"
        type="text"
        value=""
      />
    </div>
  </form>
</div>
`;
